test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3372,6 +3372,48 @@ def forward(self, x, k):
         k = torch.tensor(3)
         self.run_test(MyModuleDynamic(), [x, k])
 
+    @disableScriptTest()
+    def test_auto_grad(self):
+        class MyClip(torch.autograd.Function):
+            @staticmethod
+            def forward(ctx, input, min):
+                ctx.save_for_backward(input)
+                return input.clamp(min=min)
+
+        class MyRelu(torch.autograd.Function):
+            @staticmethod
+            def forward(ctx, input):
+                ctx.save_for_backward(input)
+                return input.clamp(min=0)
+
+        from torch.onnx.symbolic_helper import _unimplemented
```

Comment:
Please move import to the top of the file. All imports should be at the top of the file unless there's a functional reason to avoid it.